AA555A0000000000.json

```diff
@@ -0,0 +1,18 @@
+{
+  "gateway_conf": {
+    "gateway_ID": "AA555A0000000000",
```

Comment:
I don't think that's the real id of the gateway. Gateway id is based on mac address. Please follow the instructions in the wiki.